feg/gateway/policydb/policydb_test.go

```diff
@@ -181,7 +177,7 @@ func TestPolicyDBBaseNamesWithGRPC(t *testing.T) {
 
 func TestPolicyDBRulesWithGRPC(t *testing.T) {
 	onceTestsInit.Do(func() { initOnce(t) })
-	streamer_test_init.StartNewTestProvider(t, &mockStreamProvider{Name: "policydb"})
+	streamer_test_init.StartNewTestProvider(t, &mockStreamProvider{Name: "policydb"}, "policydb")
```

Comment:
`mockStreamProvider` doesn't need to know what its stream name is, so can remove it altogether from the object defn